src/OpenTelemetry.Exporter.Geneva/Metrics/TlvMetricExporter.cs

```diff
@@ -238,6 +238,11 @@ internal ExportResult Export(in Batch<Metric> batch)
             }
         }
 
+        if (result == ExportResult.Success)
```

Comment:
We can skip logging this for Metrics. It's already there in SDK:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/1bbafaa7b7bed6470ff52fc76b6e881cd19692a5/src/OpenTelemetry/Metrics/BaseExportingMetricReader.cs#L87-L97

Reply:
I have removed the logging from metrics